server/services/licenceService.ts

```diff
@@ -485,8 +505,9 @@ export class LicenceService {
 
```

Comment:
This is where the vary address form data is used to update the licence.  Before doing that generate a version of fieldConfig that selects the address use by pdfFormatter